frontend/src/components/InitialSetup.vue

```diff
@@ -88,7 +88,7 @@
                   <input id="confirmSetupKey" v-model="confirmSetupKey" name="confirmSetupKey" type="checkbox" class="h-4 w-4 rounded border-gray-300 text-primary focus:ring-primary" required>
```

Comment:
I noticed a "Setup Key" here. :smile:

Reply:
Good catch. Impressive how many variations of this term we had.